.github/workflows/aks.yml

```diff
@@ -17,7 +17,7 @@ on:
 env:
   GOARCH: amd64
   CGO_ENABLED: 0
-  SETUP_GO_VERSION: '^1.19'
+  SETUP_GO_VERSION: '1.19.*'
```

Comment:
Imho it would be better to use the version retrieve from the go.mod file 
https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file

Reply:
Interesting discussion for master, but not here. I only want to avoid newer versions that we currently see with 1.20 on some github action caches. 

Note: It's totally fine to build a 1.19 go.mod file with 1.20, it's just the version of the file format.
However, the go version in CI should be synced with the version in `Dockerfile.dapper` (which we use for the release build).